docs/getting-started-guides/gce.md

```diff
@@ -29,6 +34,16 @@ or
 wget -q -O - https://get.k8s.io | bash
 ```
 
+Once this command completes, you will have a master VM and four worker VMs, running as a Kubernetes cluster. By default, some containers will already be running on your cluster. These are used to run and monitor Kubernetes.
+
+If you run into trouble please see the section on [troubleshooting](https://github.com/brendandburns/kubernetes/blob/docs/docs/getting-started-guides/gce.md#troubleshooting), or come ask questions on IRC at #google-containers on freenode.
```

Comment:
You should link to the relative path within this repo (or at worst to the GCP repo), not to Brendan's repo that may go away. I think that `docs/getting-started-guides/gce.md` should work as a relative path, but you may have to play around with it a little since I don't quite remember.